index.php

```diff
@@ -65,6 +65,24 @@ function getLastRevision($distro) {
 Jan 19th 2019 - Artful jobs disabled (but packages still available)<br />
 </div>
 
+<div class="rel_section">Automatic installation script</div>
+For convenience there is an automatic installation script available that installs LLVM for you.
+
+<div class="rel_boxtext">
+To install the latest stable version:
+<pre>
+sh -c "$(wget -O - http://apt.lvm.org/llvm.sh)"
```

Comment:
A typo? Should it be `apt.llvm.org`?

Also, could we recommend using `https` instead?
Executing anything that came from `http` is just too risky.